pom.xml

```diff
@@ -36,5 +36,6 @@
     <module>sdk/template</module>
     <module>sdk/textanalytics</module>
     <module>sdk/spring</module>
+    <module>sdk/spring/azure-spring-cloud-test-storage</module>
```

Comment:
There's an accumulating pom file in sdk/spring path, and this test module should be added there.

Reply:
done